login-Example/login-Example/AppDelegate.swift

```diff
@@ -42,7 +42,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         SimpleResolver.sharedResolver.store(factory: Factory(type: InfomaniakLoginable.self) { _, _ in
```

Comment:
I'ts a nice to have, but we could migrate this to Tuist, so we have the library in one hand, and a managed example project in the other. What do you think ?

Reply:
Originally the intent of this project was that anyone outside Infomaniak could easily run the project and understand it without additional setup. However with time the project evolved to something to test the login instead of an example. I think it would be better to add tests to the framework and simplify the example.